src/sass/main.css

```diff
@@ -0,0 +1,334 @@
+  .header-img__circle {
+    background-image: url("../images/mobile/header-circle-568.png");
+    background-size: 284px 344px;
+  }
+}
+
+.slogan {
+  width: 230px;
+  margin-bottom: 19px;
+}
+
+.slogan__title {
+  color: var(--color-white);
+  font-family: "Titan One", cursive;
+  font-weight: 400;
+  font-size: 26px;
+  line-height: 1.15;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+}
+.slogan__title--accent {
+  color: var(--color-header);
+  font-size: 48px;
+  line-height: 0.96;
+}
+
+.buttons-header {
+  display: flex;
+  gap: 8px;
+}
+
+.buttons-header__btn {
+  height: 24px;
+  border-radius: 20px;
+  border: none;
+  color: var(--color-white);
+  font-size: 8px;
+  line-height: 1.25;
+  background-color: var(--color-header);
+  padding-inline: 14px;
+  padding-block: 7px;
+  cursor: pointer;
+  transition: background-color 250ms;
+}
+.buttons-header__btn:hover, .buttons-header__btn:focus {
+  background-color: #c7133e;
+}
+.buttons-header__btn--color-change {
+  color: var(--color-header);
+  background-color: var(--color-white);
+}
+.buttons-header__btn--color-change:hover, .buttons-header__btn--color-change:focus {
+  background-color: #ecebeb;
+}
+
+.mobile-menu {
+  background-color: var(--color-mobile-menu);
+  position: fixed;
+  top: 0px;
+  right: 0;
+  z-index: 10;
+  overflow-y: scroll;
+  width: 250px;
+  height: 100%;
+  padding-top: 60px;
+  text-align: center;
+  transform: translateX(100%);
+  transition: transform 0.5s ease-in-out;
+}
+.mobile-menu.is-open {
+  transform: translateX(0);
+}
+
+.mobile-menu__toggle {
+  width: 20px;
+  height: 12px;
+  background-color: var(--color-ice-cream);
+  border: none;
+  cursor: pointer;
+}
+.mobile-menu__toggle--makeover {
+  width: 14px;
+  height: 14px;
+  position: absolute;
+  top: 23px;
+  right: 28px;
+  background-color: var(--color-mobile-menu);
+}
+
+.mobile-menu__btn {
+  width: 209px;
+  position: relative;
+  margin-bottom: 19px;
+  transition: background-color 250ms;
+}
+
+.mobile-menu__icon {
+  position: absolute;
+  top: 18px;
+  right: 46px;
+}
+
+.mobile-nav {
+  text-align: start;
+}
+
+.mobile-nav__item {
+  border-top: 1px solid #e18298;
+  padding-block: 12px;
+  padding-left: 21px;
+}
+.mobile-nav__item:last-child {
+  border-bottom: 1px solid #e18298;
+}
+
+.mobile-nav__link {
+  color: var(--color-white);
+  font-weight: 700;
+  font-size: var(--font-14px);
+  line-height: 1.29;
+  transition: color 250ms;
+}
+.mobile-nav__link:hover, .mobile-nav__link:focus {
+  color: var(--color-header);
+}
+
+.button {
+  height: 44px;
+  border-radius: 22px;
+  cursor: pointer;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 1.31;
+  border: none;
+}
+.button--bright {
+  color: #d41443;
+  background-color: #fff;
+}
+.button--bright:hover {
+  background-color: #ecebeb;
+}
+.button--dark {
+  color: #fff;
+  background-color: #d41342;
+}
+.button--dark:hover {
+  background-color: #c7133e;
+}/*# sourceMappingURL=main.css.map */
```

Comment:
Co tu się ...?

Reply:
tego nie ruszałem

Reply:
a jeśli coś dodało w sumie nie wiem jak to nie merguj tego tylko to olej